manifest.json

```diff
@@ -81,5 +81,13 @@
     "projectOptions": ["app", "addon"],
     "nodeVersion": "6.0.0",
     "commands": ["ember-3x-codemods cp-volatile app/**/*.js"]
+  },
+  "cp-property-map-codemod": {
+    "versions": {
+      "ember-cli": "3.9.0-beta.1"
```

Comment:
Should we target the ember-source version that introduced the new API?

Reply:
Yes I think we should, and also for the other recent additions like cp-property, deprecate-router, deprecate-merge, we should also do the same right?
Will create separate issues for the same.